content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -261,66 +416,135 @@ program to retrieve the contents of your secret data.
    kubectl get secret secret1 -n default -o yaml
    ```
 
-   The output should contain `mykey: bXlkYXRh`, with contents of `mydata` encoded, check
+   The output should contain `mykey: bXlkYXRh`, with contents of `mydata` encoded using base64;
+   read
    [decoding a Secret](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#decoding-secret)
-   to completely decode the Secret.
+   to learn how to completely decode the Secret.
+
+### Ensure all relevant data are encrypted {#ensure-all-secrets-are-encrypted}
 
-## Ensure all Secrets are encrypted
+For this example, you have configured your cluster so that Secrets are encrypted on write.
+Performing a replace operation for each Secret will encrypt that content at rest,
+where the objects are unchanged.
 
-Since Secrets are encrypted on write, performing an update on a Secret will encrypt that content.
+You can make this change across all Secrets in your cluster:
 
 ```shell
+# Run this as an administrator that can read and write all Secrets
 kubectl get secrets --all-namespaces -o json | kubectl replace -f -
 ```
 
-The command above reads all Secrets and then updates them to apply server side encryption.
+The command above reads all Secrets and then updates them with the same data, in order to
+apply server side encryption.
 
 {{< note >}}
 If an error occurs due to a conflicting write, retry the command.
-For larger clusters, you may wish to subdivide the secrets by namespace or script an update.
+It is safe to run that command more than once.
+
+For larger clusters, you may wish to subdivide the Secrets by namespace,
+or script an update.
 {{< /note >}}
 
-## Rotating a decryption key
+## Cleanup {#cleanup-all-secrets-encrypted}
+
+{{< warning >}}
+Making this change prevents the API server from retrieving resources that are marked
```

Comment:
nit: Clean up <what>
nit: What is the change that the reader is making?

Reply:
It is: remove the `identity` part of the encryption configuration. The heading could be more obvious.

Reply:
How's the new wording here?